sonarqube/README.md

```diff
@@ -13,6 +13,50 @@ No additional installation is needed on your server.
 
 ### Configuration
 
+SonarQube exposes metrics from two sources: its web API and JMX. To collect all of the
+[metrics specified below](#metrics), you'll need to configure two intances of this check. One instance will monitor
+SonarQube's JMX beans and the other will monitor SonarQube's Web API.
+
+Documentation on SonarQube's Web API is available at `/web_api` on your SonarQube web UI. By default this integration
+collects all relevant SonarQube performance metrics exposed through SonarQube's JMX beans. The configuration for these
+default metrics is available in the [sonarqube.d/metrics.yaml][12] file. Documentation on these beans is available on
```

Comment:
should the link text here match the filename on github, which is `sonarqube/data/metrics.yaml`?

Reply:
When installed its right next to the `conf.yaml` -- the `data` directory doesn't exist

Reply:
```
vagrant@sonarqube:~$ ll /etc/datadog-agent/conf.d/sonarqube.d/
-rw-r--r--   1 dd-agent dd-agent  1537 Mar 11 19:29 conf.yaml
-rw-r--r--   1 dd-agent dd-agent 21775 Feb 23 11:28 conf.yaml.example
-rw-r--r--   1 dd-agent dd-agent  2688 Feb 23 11:28 metrics.yaml
```